doc/source/whatsnew/v0.18.2.txt

```diff
@@ -51,6 +51,53 @@ New behaviour:
 
    In [2]: pd.read_csv(StringIO(data), names=names)
 
+.. _whatsnew_0182.enhancements.semi_month_offsets:
+
+Semi-Month Offsets
+^^^^^^^^^^^^^^^^^^
+
+Pandas has gained new frequency offsets, ``SemiMonthEnd`` ('SM') and ``SemiMonthBegin`` ('SMS').
+These provide date offsets anchored (by default) to the 15th and end of month, and 15th and 1st of month respectively.
+Optionally you can specify the ``day_of_month`` which will be used instead of the 15th each month. Anchoring suffixes
+are also supported for setting the ``day_of_month``.
+(:issue:`1543`)
+
+.. ipython:: python
+
+    from pandas.tseries.offsets import SemiMonthEnd, SemiMonthBegin
+
+SemiMonthEnd:
+
+.. ipython:: python
+
+    Timestamp('2016-01-01') + SemiMonthEnd()
+
+    pd.date_range('2015-01-01', freq='SM', periods=4)
+
+SemiMonthBegin:
+
+.. ipython:: python
+
+    Timestamp('2016-01-01') + SemiMonthBegin()
+
+    pd.date_range('2015-01-01', freq='SMS', periods=4)
+
+Using the keyword argument ``day_of_month``:
+
+.. ipython:: python
+
+    Timestamp('2016-01-01') + SemiMonthBegin(day_of_month=16)
```

Comment:
No need to show this, it is essentially an implementation detail (the ``day_of_month`` one), the anchor is much more important.